mathesar_ui/src/sections/table-view/header/HeaderCell.svelte

```diff
@@ -3,8 +3,10 @@
     faSortAmountDown,
     faSortAmountDownAlt,
     faThList,
+    faSpinner,
   } from '@fortawesome/free-solid-svg-icons';
-  import { Dropdown, Icon } from '@mathesar-components';
+  import { Checkbox, Dropdown, Icon } from '@mathesar-components';
+  import type { ConstraintsDataStore } from '@mathesar/stores/table-data/constraints';
```

Comment:
We can import the types from the type definition file.

```suggestion
  import type { ConstraintsDataStore } from '@mathesar/stores/table-data/types';
```

Reply:
@seancolsen This is still present in some files like Header.svelte, but I think this can be handled in one of your later PRs.

Reply:
Whoops, sorry about that. I misunderstood and thought you were only pointing out to one file that needed to be fixed so I didn't look elsewhere. I've fixed others in b92be5be7ff0358847e93a405672b4d6511fa30d